src/components/common/bubble/bubble.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as styles from './bubble.module.scss';
+import classNames from 'classnames';
+
+type Props = {
+  size?: 'small' | 'regular';
+};
+const Bubble: React.FC<React.PropsWithChildren<Props>> = ({
```

Comment:
Define children in props
```suggestion
const Bubble: React.FC<Props> = ({
```